packages/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt

```diff
@@ -708,7 +704,7 @@ class ExpoCameraView(
   }
 
   override fun setPreviewTexture(surfaceTexture: SurfaceTexture?) {
-    glSurface = surfaceTexture
+    glSurfaceTexture = surfaceTexture
```

Comment:
Would the order of execution be an issue here? I'm thinking about if `setPreviewTexture` was called after `onViewAdded` we wouldn't have a texture to set on the preview.

Reply:
It should fine. It's called when `createCameraTextureAsync` is called from js. Then we will rebuild the camera preview